iis-wf/iis-wf-top/src/test/resources/eu/dnetlib/iis/wf/top/integration/import/oozie_app/workflow.xml

```diff
@@ -34,6 +34,15 @@
 					<name>output</name>
 					<value>${workingDir}/mr_import</value>
 				</property>
+               <property>
```

Comment:
Yeah, I stuck on the very same issue this morning while testing importer integration test after moving it to `iis-wf-import` module.

This means integration test was never run on CDH5 infrastructure as it requires providing zookeeper properties explicitly. CDH4 cluster was configured in a way we didn't have to bother about those values, it was taken from environment which is much easier and convenient.